servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H2ToStH1ClientDuplexHandler.java

```diff
@@ -63,7 +63,7 @@ final class H2ToStH1ClientDuplexHandler extends AbstractH2DuplexHandler {
     H2ToStH1ClientDuplexHandler(boolean sslEnabled, BufferAllocator allocator, HttpHeadersFactory headersFactory,
                                 CloseHandler closeHandler, StreamObserver observer) {
         super(allocator, headersFactory, closeHandler, observer);
-        this.scheme = sslEnabled ? HttpScheme.HTTPS : HttpScheme.HTTP;
+        this.scheme = sslEnabled ? HttpScheme.https() : HttpScheme.http();
```

Comment:
It's preferable to keep Netty's `HttpScheme` here to take advantage of `AsciiString`